content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
+    {{< /text >}}
 
-    On the VM/external host:
+4.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs`.
 
     {{< text bash >}}
-    $ host istio-pilot.istio-system
-    {{< /text >}}
 
-    Example generated message:
+    $ sudo mkdir /etc/certs
+    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
 
-    {{< text plain >}}
-    $ istio-pilot.istio-system has address 10.150.0.6
     {{< /text >}}
 
-    Check that you can resolve cluster IPs. The actual address will depend on your deployment.
+6.  Install `cluster.env` under `/var/lib/istio/envoy`.
 
     {{< text bash >}}
-    $ host istio-pilot.istio-system.svc.cluster.local.
-    {{< /text >}}
 
-    Example generated message:
+    $ sudo cp cluster.env /var/lib/istio/envoy
 
-    {{< text plain >}}
-    istio-pilot.istio-system.svc.cluster.local has address 10.63.247.248
     {{< /text >}}
 
-    Check istio-ingress similarly:
+5.  Transfer ownership of the files in `/etc/certs/` and `/var/lib/istio/envoy` to the Istio proxy.
 
     {{< text bash >}}
-    $ host istio-ingress.istio-system.svc.cluster.local.
-    {{< /text >}}
-
-    Example generated message:
 
-    {{< text plain >}}
-    istio-ingress.istio-system.svc.cluster.local has address 10.63.243.30
-    {{< /text >}}
+    $ sudo chown -R istio-proxy /etc/certs /var/lib/istio/envoy
 
-*   Verify connectivity by checking whether the VM can connect to Pilot and to an endpoint.
-
-    {{< text bash json >}}
-    $ curl 'http://istio-pilot.istio-system:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
-    {
-      "hosts": [
-       {
-        "ip_address": "10.60.1.4",
-        "port": 8080
-       }
-      ]
-    }
     {{< /text >}}
 
-    On the VM, use the address above. It will directly connect to the pod running istio-pilot.
+7.  Verify the node agent works:
 
     {{< text bash >}}
-    $ curl 'http://10.60.1.4:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
-    {{< /text >}}
 
-*   Extract the initial Istio authentication secrets and copy them to the machine. The default
-installation of Istio includes Citadel and will generate Istio secrets even if
-the automatic `mTLS`
-setting is disabled (it creates a secret for each service account, and the secret
-is named as `istio.<serviceaccount>`). It is recommended that you perform this
-step to make it easy to enable mutual TLS in the future and to upgrade to a future version
-that will have mutual TLS enabled by default.
+    $ sudo node_agent
+    ....
+    CSR is approved successfully. Will renew cert in 1079h59m59.84568493s
 
-    `ACCOUNT` defaults to 'default', or `SERVICE_ACCOUNT` environment variable
-    `NAMESPACE` defaults to current namespace, or `SERVICE_NAMESPACE` environment variable
-    (this step is done by machineSetup)
-    On a Mac either `brew install base64` or `set BASE64_DECODE="/usr/bin/base64 -D"`
+    {{< /text >}}
+
+8.  Start Istio using `systemctl`.
```

Comment:
To distinguish from files and folders, I recommend you identify `systemctl` as a command:
```suggestion
8.  Start Istio using the `systemctl` command.
```

Reply:
Why would the user think it was a file or folder?